.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
move files genetrated by editors to global .gitignore: https://help.github.com/articles/ignoring-files/